doc/radosgw/federated-config.rst

```diff
@@ -817,10 +817,7 @@ there is a unified namespace between the two regions.
 
 .. _CRUSH Map: ../../rados/operations/crush-map
 .. _Install Ceph Object Gateway: ../../install/install-ceph-gateway
-.. _Cephx Administration: ../../rados/operations/authentication/#cephx-administration
-.. _Ceph configuration file: ../../rados/configuration/ceph-conf
```

Comment:
@tchaikov  I dropped these changes since they are irrelevant. Are the others look good now?